common/src/main/java/com/wynntils/models/emeralds/EmeraldModel.java

```diff
@@ -191,7 +201,13 @@ public void onContainerSetContent(ContainerSetContentEvent.Pre event) {
         }
 
         for (int i = 0; i < containerStop; i++) {
-            adjustBalance(items.get(i), 1, false);
+            ItemStack is = items.get(i);
```

Comment:
```suggestion
            ItemStack itemStack = items.get(i);
```

We like to use itemStack name, instead of is, item, stack, etc so the code is overall more readable in Artemis.